src/providers/krb5/krb5_child_handler.c

```diff
@@ -422,6 +422,18 @@ errno_t set_extra_args(TALLOC_CTX *mem_ctx, struct krb5_ctx *krb5_ctx,
         }
     }
 
+    if (krb5_ctx->check_pac) {
+        extra_args[c] = talloc_asprintf(extra_args,
```

Comment:
Shouldn't size of `extra_args` be adjusted?

Reply:
fixed